### PR TITLE
Workaround for adding an issue to a sprint

### PR DESCRIPTION
**Pull request by dann7387**
Fixes Issue #109.

### CODE REVIEW

File: jira/client.py [hunk 1]
```diff
@@ -2692,9 +2693,11 @@ def add_issues_to_sprint(self, sprint_id, issue_keys):
         :param issue_keys: the issues to add to the sprint
         """
         data = {}
-        data['issueKeys'] = issue_keys
-        url = self._get_url('sprint/%s/issues/add' %
-                            (sprint_id), base=self.AGILE_BASE_URL)
+        data['idOrKeys'] = issue_keys
+        data['customFieldId'] = 10004
```
Comment (dbaxa) [inline jira/client.py]: The `customFieldId` should not be a hard-coded constant.

**Comment by dann7387** [comment]
@dbaxa The code will now grab the entire list of fields and finds the field with the name "Sprint".

Let me know if you see anything else.

**Comment by dbaxa** [comment]
Please add a test for  this change.

**Comment by asqui** [comment]
FYI: I'm going to submit a PR to add some helpers for mapping custom fields to their IDs (and the reverse)

You may also find it easier to use the JIRA.fields() method to get the field metadata, rather than constructing and fetching the URL manually...

**Comment by dann7387** [comment]
Code updated to use JIRA.fields().
Unit tests added.

**Pull request merged**